src/Sylius/Bundle/CurrencyBundle/Resources/config/serializer/Model.ExchangeRate.yml

```diff
@@ -9,8 +9,11 @@ Sylius\Component\Currency\Model\ExchangeRate:
         ratio:
             expose: true
             type: float
+            groups: [Default, Detailed]
         sourceCurrency:
             expose: true
+            groups: [Default, Detailed]
         targetCurrency:
             expose: true
+            groups: [Default, Detailed]
```

Comment:
Having `updatedAt` field on `ExchangeRate` model could be useful, but that's something that can be added later as well 😉